### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by RustyBl0ck** [Kitteh6660/DramaticDoors#100]
**Important Information**
Minecraft Version: 1.19.2
Mod Loader System: Fabric 0.15.2
DramaticDoors Version: 3.1.5

**Describe the bug**
When uninstalling the mod from server, all blocks (with the exception of a few) are replaced with other blocks. Attached you will see what its supposed to look like vs what is actually looks like.

**To Reproduce**
1. Install DD on Fabric (server and client).
2. Uninstall DD on server
3. Restart server
4. Open server through client

**Expected behavior**
All blocks should be rendered as intended, not replaced randomly.

**Screenshots**
<img width="3360" alt="Screenshot 2024-08-01 at 9 25 40 PM" src="https://github.com/user-attachments/assets/5f2d5dd3-149d-4fcb-ba3d-068d6bddc079">With DD Installed

![Screenshot 2024-08-01 at 9 09 59 PM](https://github.com/user-attachments/assets/cec7a88e-a66e-43cb-bb74-ac5517a4b3e4)With DD Uninstalled

**Comment by Kitteh6660** [comment]
This is the result of mismatched waterlogging setting.

Try turning off waterlogging from Dramatic Doors config and the blocks should be correct.

**Comment by RustyBl0ck** [comment]
Waterlogging was on before for both doors and gates. Keep in mind that the server looks fine WITH DD installed. So, with DD installed, I tried disabling the waterlogging and restarted the server. It now looks like the second image I sent with waterlogging disabled...

**Comment by RustyBl0ck** [comment]
TL;DR: I broke the server again when disabling waterlogging. I'd like to completely uninstall DD without breaking the server...

**Comment by Kitteh6660** [comment]
The settings need to match both on client and the server.
